generation_config.yaml

```diff
@@ -0,0 +1,25 @@
+gapic_generator_version: 2.41.0
+googleapis_commitish: bcaed39fd1a805a6411a3992ea32dc1ba0ba7ec3
+libraries_bom_version: 26.38.0
+libraries:
+  - api_shortname: datastore
+    name_pretty: Cloud Datastore
+    product_documentation: https://cloud.google.com/datastore
+    client_documentation: https://cloud.google.com/java/docs/reference/google-cloud-datastore/latest/history
+    issue_tracker: https://issuetracker.google.com/savedsearches/559768
+    release_level: stable
+    language: java
+    repo: googleapis/java-datastore
+    repo_short: java-datastore
+    distribution_name: com.google.cloud:google-cloud-datastore
+    codeowner_team: @googleapis/cloud-native-db-dpes @googleapis/api-datastore-sdk @googleapis/api-firestore-partners
```

Comment:
We may need a single quote for this field, see https://github.com/googleapis/google-cloud-java/blob/main/generation_config.yaml#L221